rebar.config

```
@@ -161,6 +163,7 @@
         {relx, [
             {dev_mode, false},
             {include_erts, true},
```

{include_erts, false}
```suggestion
            {include_erts, true},
```